content/07.pathogenesis.md

```diff
@@ -10,12 +10,20 @@ This section would also be great for the introduction of zoonotic diseases which
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
 
-#### Viral Replication, Spreading and Transmission
+#### Viral Replication and Spreading
 
+[Renamed section with transmission dynamics below] 
 [Basic introduction into replication cycle]
-[What is the basic reproductive rate]
 [What are the routes of transmission]
 
+#### Reproduction Number and Dynamics of Transmission
+
+Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions. The basic reproduction number, R0, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. The effective reproduction number, R(t), describes how the reproduction number may change over time, and is used to quantify deviations in R from R0, for example as some fraction of the population becomes infected, or as interventions are put into place. R0 and R(t) can be estimated directly from epidemiological data or inferred using mathematical modeling, the latter is typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74]. 
```

Comment:
```suggestion
Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions. 
The basic reproduction number, R0, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 
The effective reproduction number, R(t), describes how the reproduction number may change over time, and is used to quantify deviations in R from R0, for example as some fraction of the population becomes infected, or as interventions are put into place. 
R0 and R(t) can be estimated directly from epidemiological data or inferred using mathematical modeling, the latter is typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74]. 
```
Although it's weird to get used to, it actually works out quite nicely to put one sentence per line (because then it's easy to see which sentences have been changed when looking at the full manuscript). They will show up as a coherent paragraph in the text.